### PR TITLE
Cloud build configuration for cloud-provider-aws

### PR DESCRIPTION
**Pull request by nckturner**
Creating a cloudbuild job for image pushes. [PR for cloudbuild configuration](https://github.com/kubernetes/cloud-provider-aws/pull/138)

### CODE REVIEW

**Comment by spiffxp** [comment]
please fix tests

File: config/jobs/image-pushing/k8s-staging-cloud-provider-aws.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+postsubmits:
+  # This is the github repo we'll build from. This block needs to be repeated for each repo.
+  kubernetes/cloud-provider-aws:
+    - name: cloud-provider-aws-push-images
+      cluster: k8s-infra-prow-build-trusted
+      annotations:
+        testgrid-dashboards: sig-aws-cloud-provider-aws-image-pushes
```
Comment (wongma7) [inline config/jobs/image-pushing/k8s-staging-cloud-provider-aws.yaml:7]: also add to testgrid? e.g. https://github.com/kubernetes/test-infra/pull/18511/files

**Comment by wongma7** [comment]
There is no cloudbuild.yaml in https://github.com/kubernetes/cloud-provider-aws yet. Can you link to the PR to add that and merge that before we merge this.

**Comment by randomvariable** [comment]
> There is no cloudbuild.yaml in kubernetes/cloud-provider-aws yet. Can you link to the PR to add that and merge that before we merge this.

The cloudbuild config is current in progress at https://github.com/kubernetes/cloud-provider-aws/pull/138

**Comment by randomvariable** [comment]
Other than needing to add the dashboard, the approach looks correct. @wongma7 should get get the cloudbuild PR merged in first then?

File: config/jobs/image-pushing/k8s-staging-cloud-provider-aws.yaml (hunk 1 above)
Comment (nckturner) [reply config/jobs/image-pushing/k8s-staging-cloud-provider-aws.yaml:7]: Added to PR

**Comment by nckturner** [comment]
@wongma7 Actually using creating the dashboard in this PR didn't work because (I think) the dashboard needs to exist already in order to pass validation, but also I realized its kinda dumb to have a dashboard specific to cloud-provider-aws image pushes, so I used a new tab in the existing sig-aws-cloud-provider-aws dashboard.

**Review by fejta — APPROVED** [review]

**Pull request merged**